### DISCUSSION TITLE
Nextjs lazyBoundary not working on Marquee?

### DISCUSSION THREAD

**Discussion by Github010101011** [vercel/next.js#37202] [Help]
I've reviewed issue: https://github.com/vercel/next.js/issues/32774

I'm using the library React-Fast-Marquee.  To optimize it, I'm using nextjs images. The issue is that: the images load too slow and is blank for a while/period of time before it pops up.  Most noticeable in mobile view.

I've tried using lazyBoundary with LazyRoot, but they're both not working.

**Comment by raymondkneipp** [comment]
I had this same issue. I was able to fix the problem by adding `priority={true}` to all the next.js images inside to marquee. https://nextjs.org/docs/api-reference/next/image#priority
```
<Marquee>
    <Image src="..." layout="fixed" width="100" height="100" priority={true} />
    <Image src="..." layout="fixed" width="100" height="100" priority={true} />
    ...
</Marquee>
```

**Comment by MadMaxMcKinney** [comment]
I'm also having this issue, and adding the priority flag doesn't fix the root of the issue.

On mobile a marquee element can be completely on-screen and the images will not load (even svg's), they will only load when interacted with in some way.

**Comment by JoydebCrowdVolt** [comment]
Render A Skeleton Loader and then use the onMount prop of Marquee which takes in a callback function which you can use to render the marquee and hide the skeleton and show the marquee, it helps in prevented the blink or the uneasy load of inner cards or images and also for smooth image load you can use the onLoad prop of Next.js Image component.

Hope it works!
God bless
